README.md

```diff
@@ -1,128 +1,105 @@
 [![Griptape Discord](https://dcbadge.vercel.app/api/server/gnWRz88eym?compact=true&style=flat)](https://discord.gg/gnWRz88eym)
 
-CLI for the Griptape Framework and Cloud.
+The Griptape CLI is a command-line interface for interacting with features of [Griptape Cloud](https://www.griptape.ai/cloud).
+Today, it provides an emulator for Griptape Cloud Skatepark, which allows you to run and test your Structures locally. 
 
 ## Prerequisites
+    - A GitHub account
+    - Python 3.9 or later
 
-CLI `cloud` commands require a [Griptape Cloud](https://www.griptape.ai/griptape-cloud/) account.
-
-## Quick Start
+## Installation 
 
 1. Install griptape-cli
-   ```
-   pipx install griptape-cli
-   ```
-1. Verify installation
-   ```
-   gt --help
-   ```
-
-## Griptape Cloud
-
-### Skatepark Emulator
+    ```bash
+    pipx install griptape-cli
+    ```
+
+   You can also install a pre-release version from GitHub.
+    ```bash
+    git clone https://github.com/griptape-ai/griptape-cli.git
+    ```
+
+    ```bash
+    pipx install ./griptape-cli
+    ```
+2. Verify installation
+    ```bash
+    gt --help
+    ```
+
+## Skatepark Emulator
 You can use the CLI to spin up a local emulator of the Griptape Cloud Skatepark. This is useful for testing and development.
 
-1. Start by creating a new directory and navigating to it.
-```
-mkdir skatepark-local 
-cd skatepark-local
-```
-
-2. Create a `structure.py` file with with the following sample code.
-```python
-import os
-import sys
-
-from griptape.config import StructureConfig, StructureGlobalDriversConfig
-from griptape.drivers import (
-    GriptapeCloudEventListenerDriver,
-    OpenAiChatPromptDriver,
-)
-from griptape.events import (
-    EventListener,
-)
-from griptape.structures import Agent
-from griptape.tools import Calculator
-
-structure = Agent(
-    tools=[Calculator(off_prompt=False)],
-    config=StructureConfig(
-        global_drivers=StructureGlobalDriversConfig(
-            prompt_driver=OpenAiChatPromptDriver(
-                model="gpt-4",
-                stream=True,
-            ),
-        )
-    ),
-    event_listeners=[
-        EventListener(
-            driver=GriptapeCloudEventListenerDriver(
-                base_url="http://127.0.0.1:5000",
-                api_key=os.environ["GRIPTAPE_CLOUD_API_KEY"],
-            ),
-        )
-    ],
-)
-
-structure.run(sys.argv[1]) # Structure Run arguments are passed in via standard input. 
-```
-
-And a `requirements.txt` file with the following content.
-```
-griptape
-```
-
+1. Start by creating a new repository from the [Managed Structure Template](https://github.com/griptape-ai/managed-structure-template).
+    a. Make sure you're logged in to GitHub
+    b. Go to the [Managed Structure Template repo](https://github.com/griptape-ai/managed-structure-template)
+    c. Click *Use this template*
+    d. Choose *Create a new repository*
+2. Clone the repository and navigate to the root directory.
```

Comment:
"navigate to the root directory" might be misinterpreted. Can we "un-ask" this question and the remedial "how to clone things" steps by changing step 2 to:
2. Clone your newly-created repository to a directory in your local development environment so that you can begin authoring your own Griptape Managed Structure.

(then skip all the git clone and cd stuff)

Reply:
Updated step 4 too to emphasize that if you're copy pasting these commands they need to be within the repo directory.